tg/controllers/restcontroller.py

```diff
@@ -68,5 +68,9 @@ class RestController(DecoratedController, CoreDispatcher, RestDispatcher):
       `CrudRestController <../main/Extensions/Crud/index.html>`_  A way to integrate ToscaWdiget Functionality with RESTful Dispatch.
 
     """
+    @expose()
+    def options(self):
+        return ""
```

Comment:
Should return `None`, so that the response is a `204 No Content` instead of being a `200 OK` with no content.